packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -36,6 +36,8 @@
     "mutedMessage": "You're muted",
     "networkReconnectMoreDetails": "Looks like something went wrong. We're trying to get back into the call.",
     "networkReconnectTitle": "Hold on",
+    "deniedPermissionToRoomDetails": "You were not invited to this room.",
```

Comment:
I would update the strings here (and the line below) too, we can't know exactly that they weren't "invited" to the room and instead use a more generic "You do not have permission to join this room"